### DISCUSSION TITLE
Is there an option to track people?

### DISCUSSION THREAD

**Discussion by jhavens12** [MartinPham/FindMySync#21] [General]
I saw it mentioned elsewhere, but don't see it listed here anywhere or within the app.

Thanks

**Comment by mattleiphamellis** [comment]
It appears Martin did leave [a comment](https://www.reddit.com/r/homeassistant/comments/10cwxxn/comment/jdx3yo1/?utm_source=share&utm_medium=web2x&context=3) on a Reddit thread last March indicating recently added People support, the current source code only seems to locate and parse Devices.data and Items.data.
